task/thermal/src/control.rs

```diff
@@ -913,17 +985,25 @@ impl<'a> ThermalControl<'a> {
         last_err.map_err(|_| ThermalError::DeviceError)
     }
 
-    /// Sets the PWM for a single fan
+    /// Sets the PWM for a single fan if it is present
```

Comment:
This has different semantics than `set_pwm`, which sets a PWM of 0 for a missing fan; does this make sense?

Reply:
It does not. Fixing!